cleanup/README.md

```diff
@@ -0,0 +1,14 @@
+# Setup environment
+
+Action cleans up workspace directory after previous run. The main reason to add
```

Comment:
In other places you don't use `up`. Either to remove it from here or add to everywhere.

Reply:
done